tests/pyret/tests/test-statistics.arr

```diff
@@ -89,3 +113,12 @@ check "linear regression":
 
 end
 
+check "multiple regression":
+  x-s-s = [list: {4}, {4.5}, {5}, {5.5}, {6}, {6.5}, {7}]
+  y-s   = [list: 33, 42, 45, 51, 53, 61, 62]
+  pf = multiple-regression(x-s-s, y-s)
+  pf({8}) is%(within-abs(0.001)) 73.3214
```

Comment:
Use `is-roughly` instead of `is%(within-abs(0.001))`, if you can get two more digits of accuracy into the test...